.gitignore

```diff
@@ -0,0 +1 @@
+aws-dual-stack/_install_
```

Comment:
Does this include the yaml files? We don't want the user to accidentally check in their modified config scripts.